### ISSUE TITLE
versioning vraptor-site docs

### ISSUE THREAD

**Issue by Turini** [caelum/vraptor4#1009]

**Comment by angeliski** [comment]
Do you like to see the version below the Download button, above the vraptor 3 link? It's only put the last version found in https://github.com/caelum/vraptor4/releases (now is v4.2.0-RC3).

**Comment by Turini** [comment]
not really. I'd like to provide different doc pages for different versions of the fw.
Currently it's not clear that some feature works only since version X

**Comment by angeliski** [comment]
Sorry, @Turini , but I can't understand your idea. You can provide one example?

**Comment by Turini** [comment]
i mean something like this:
![image](https://cloud.githubusercontent.com/assets/1529021/14593557/2294678e-0502-11e6-9947-458a6f0037b6.png)
this is a print from play framework's website. You can see the version number at the URL as well
https://www.playframework.com/documentation/2.5.x/Tutorials

**Comment by angeliski** [comment]
Nice! I will try think about one solution for that.
